.github/workflows/cut-release-branch.yml

```diff
@@ -125,20 +122,36 @@ jobs:
           message="New branch ${{ inputs.new_branch_name }} created"
           echo "::notice title="New branch created": $title::$message"
 
-      # TODO: need to add scripts into the adapters to automate bumping the core 
-      # dependency on the `*.latest` branch.  The script should also bump the 
-      # nightly release test branch to the new `*.latest` branch
-      # - name: "Bump dependencies via script"
-      #   # This bumps the dependency on dbt-core in the adapters
-      #   if: ${{ !contains(github.repository, 'dbt-core') }}
-      #   run: |
-      #     echo ${{ github.repository }}
-      #     echo "running some script yet to be written now"
-      #     scripts/update_dependencies.sh
-      #     git status
-      #     git add .
-      #     git commit -m "$commit_message"
-      #     git push
+      - name: "Bump dependencies via script"
+        # This bumps the dependency on dbt-core in the adapters
+        if: ${{ !contains(github.repository, 'dbt-core') }}
+        run: |
+          echo ${{ github.repository }}
+          echo "running update_dependencies script"
+          bash ${GITHUB_WORKSPACE}/.github/scripts/update_dependencies.sh ${{ inputs.new_branch_name }}
+          commit_message="bumping .latest branch variable in update_dependencies.sh to ${{ inputs.new_branch_name }}"
+          git status
+          git add .
+          git commit -m "$commit_message"
+          git push
+
+      - name: "Bump env variable via script"
+        # bumps the RELEASE_BRANCH variable in nightly-release.yml in adapters
+        if: ${{ !contains(github.repository, 'dbt-core') }}
+        run: |
+          file="./.github/scripts/update_release_branch.sh"
+          if test -f "$file"; then
+            echo ${{ github.repository }}
+            echo "running some script yet to be written now"
+            bash ${GITHUB_WORKSPACE}/.github/scripts/update_release_branch.sh ${{ inputs.new_branch_name }}
```

Comment:
you can use `file` here I think:
`bash "${file}" ${{ inputs.new_branch_name }}`